Intersect.Server/Networking/PacketSender.cs

```diff
@@ -955,9 +955,9 @@ public static void SendEntityDir(Entity en)
         }
 
         //EntityAttackPacket
-        public static void SendEntityAttack(Entity en, int attackTime)
+        public static void SendEntityAttack(Entity en, int attackTime, bool isBlocking = false)
```

Comment:
I think this parameter should always be explicit, no default `= false` here.

Reply:
The intention is to make blocking separate from the attack, later on (maybe after this merge), so it's implicit only for the blocking use case, to make it easier to remove when separating from the attack